README.md

```diff
@@ -44,6 +44,10 @@ BATCH_SIZE (int)
 DELETE_KEYS (str == 'True' or 'False')
 ```
 
+## Testing
+In order to run the tests, make sure you have a local Redis running.
+From there, run ```python3 /tests/tests.py```
```

Comment:
Probably `./tests/tests.py`, not the root of the fs right? 

Also might want to consider a pipenv or a dockerfile to make sure all the deps are setup & versioned right (I see you have a requirements.txt so that might be okay too).

Reply:
I think I might have to do that for CI, so we can test everything, and we need a running redis in order to test it.

Reply:
Maybe if it's a docker container or something, but for the readme it'd be helpful to point to a relative file for ppl to understand what you wanna run (ie not something on the root of their fs)

Reply:
I fixed the readme in another PR.